db_structure.sql

```diff
@@ -22,7 +22,18 @@ CREATE TABLE Users (
     id SERIAL PRIMARY KEY,
     email VARCHAR(100) NOT NULL UNIQUE,
     password VARCHAR(256) NOT NULL,
-    full_name VARCHAR(256) NOT NULL
+    full_name VARCHAR(256) NOT NULL,
+    gender VARCHAR(100),
+    birthdate DATE,
+    goals VARCHAR(100),
+    skin_types VARCHAR(100),
+    skin_concerns VARCHAR(100),
+    skin_conditions VARCHAR(100),
+    routine_products VARCHAR(100),
+    makeup BOOLEAN,
+    products_used VARCHAR(100),
+    skincare_brands VARCHAR(100),
+    makeup_brands VARCHAR(100);
```

Comment:
SQL also has the concept of enums. So if we use enums in TypeScript, let's also use Enums in our DB.

I'll let you google online and see how to use them.

Hint: we'll probably need to change the TS enum values to strings.